templates/header.html

```diff
@@ -61,5 +63,13 @@
             signOut();
         });
     }
+
+    const maintabsEl = document.querySelector('#maintabs');
```

Comment:
This is run every time users sign in and sign out right?

Reply:
Yes, the whole page is reloaded, so all page content that is specific to the user is recomputed.

Reply:
Ah, but I just caught a mistake in that the whole <script> was only included if the user is signed in.  Will fix in a sec.